proto/messages.go

```diff
@@ -58,6 +62,17 @@ const (
 	RequiredAcksLocal = 1
 )
 
+var SupportedByDriver = map[int16]SupportedVersion{
+	ProduceReqKind:          SupportedVersion{MinVersion: KafkaV0, MaxVersion: KafkaV0},
```

Comment:
why max version is 0?

Reply:
Because current version of our driver supports only basic version of protocol.  We need to update every command and after that update this map.  As an example I've added MetadataReq which now support versions up to 5.